size.sh

```diff
@@ -0,0 +1,28 @@
+BASE_PATH=dist/a11y-dialog
+
+# Build the dist files
+npm run build -- --silent
+
+# Gzip files
+gzip -9fkc $BASE_PATH.min.js > $BASE_PATH.min.js.gz
+gzip -9fkc $BASE_PATH.esm.min.js > $BASE_PATH.esm.min.js.gz
+
+# Brotli files
+brotli -9fkc $BASE_PATH.min.js > $BASE_PATH.min.js.br
+brotli -9fkc $BASE_PATH.esm.min.js > $BASE_PATH.esm.min.js.br
```

Comment:
```suggestion
brotli -8fkc $BASE_PATH.min.js > $BASE_PATH.min.js.br
brotli -8fkc $BASE_PATH.esm.min.js > $BASE_PATH.esm.min.js.br
```

`brotli -8` is probably more in line with what users are likely to get out of their hosting services, but I could take or leave this change.